### PR TITLE
docs: add documentation to routes

### PR DESCRIPTION
**Pull request by pyphilia**
Added documentation for some routes. 
Removed and renamed when necessary.

### CODE REVIEW

**Review by spaenleh — APPROVED** [review]
🏷️

File: src/services/item/plugins/itemLike/schemas.ts [hunk 1]
```diff
@@ -31,33 +32,58 @@ export const packedItemLikeSchemaRef = registerSchemaAsRef(
...
     [StatusCodes.OK]: itemLikeSchemaRef,
+    '4xx': errorSchemaRef,
   },
 };
 
 export const deleteOne = {
+  operationId: 'deleteItemLike',
+  tags: ['like'],
+  summary: 'Delete like for item',
+  description: 'Delete like for item.',
```
Comment (Defferrard) [inline src/services/item/plugins/itemLike/schemas.ts:82]: Should it be `Unlike an item` ?

**Review by Defferrard — APPROVED** [review]
LGTM !

**Review by ReidyT — APPROVED** [review]
👍

**Pull request merged**